### ISSUE TITLE
Consider uninstall hook

### ISSUE THREAD

**Issue by ryanbaer** [Rush/link-module-alias#8]
First of all, great project. Was very excited to use it and dug in on cleaning up my import statements. Sadly, couldn't get it to work with create-react-app (tests in particular), so I had to uninstall.

This is just a suggestion to add an uninstall hook (pre- or post-), which clears out the symlinks the package created. It could also perhaps remove the `_moduleAliases` from the package.json.

As an aside, if you know of a way to make it work with create-react-app, please let me know.
Thanks for your hard work!

**Comment by brian428** [comment]
Agreed. Having it clear out the symlinks it created during uninstall would be a good idea.
